src/tabs/tabs.js

```diff
@@ -138,17 +213,23 @@
         tab.appendChild(rippleContainer);
       }
 
-      tab.addEventListener('click', function(e) {
-        if (tab.getAttribute('href').charAt(0) === '#') {
-          e.preventDefault();
-          var href = tab.href.split('#')[1];
-          var panel = ctx.element_.querySelector('#' + href);
-          ctx.resetTabState_();
-          ctx.resetPanelState_();
-          tab.classList.add(ctx.CssClasses_.ACTIVE_CLASS);
-          panel.classList.add(ctx.CssClasses_.ACTIVE_CLASS);
-        }
-      });
+      if (tab.getAttribute('href').charAt(0) === '#') {
+        var href = tab.href.split('#')[1];
+        var tabId = href + '-tab';
+        tab.setAttribute('aria-controls', href);
+        tab.setAttribute('id', tabId);
```

Comment:
We shouldn't go overriding attributes like this. Developers expect if they set an id that it remain untouched. This change is *extremely* backwards incompatible.